Packs/CortexXDR/Playbooks/playbook-Cortex_XDR_-_Identity_Analytics.yml

```diff
@@ -0,0 +1,1775 @@
+        - Administrative user activities
+        - Security rules and policies changes
+        - Failed login attempt
+        - MFA failed login attempt
+        - Login attempt from an uncommon country
+        - Anomalies activities
+        - Risky users
+        - Uncommon high volume of actions
+        - Action uncommonly performed by the user
+      playbookName: Azure - User Investigation
+      type: playbook
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "35"
+    scriptarguments:
+      AzureSearchTime:
+        simple: ago(7d)
+      MfaAttemptThreshold:
+        simple: "10"
+      Username:
+        complex:
+          root: PaloAltoNetworksXDR.OriginalAlert.raw_abioc.event
+          accessor: auth_identity
+          transformers:
+          - operator: uniq
+      failedLogonThreshold:
+        simple: "20"
+    separatecontext: true
+    continueonerrortype: ""
+    loop:
+      iscommand: false
+      exitCondition: ""
+      wait: 1
+      max: 100
+    view: |-
+      {
+        "position": {
+          "x": 710,
+          "y": 1010
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: true
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "12":
+    id: "12"
+    taskid: e889fef1-f79e-4d52-8059-6447eb6ed7e1
+    type: condition
+    task:
+      id: e889fef1-f79e-4d52-8059-6447eb6ed7e1
+      version: -1
+      name: Found any malicious user activity?
+      description: Determine if the activity is malicious based on the investigation findings.
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#default#':
+      - "13"
+      "yes":
+      - "16"
+    separatecontext: false
+    conditions:
+    - label: "yes"
+      condition:
+      - - operator: isNotEmpty
+          left:
+            value:
+              complex:
+                root: AzureScriptBasedUserAgentEvents
+            iscontext: true
+          right:
+            value: {}
+        - operator: greaterThan
+          left:
+            value:
+              complex:
+                root: AzureFailLoginCount
+            iscontext: true
+          right:
+            value:
+              complex:
+                root: inputs.FailedLogonThreshold
+            iscontext: true
+        - operator: isNotEmpty
+          left:
+            value:
+              complex:
+                root: SuspiciousUserAgent
+            iscontext: true
+        - operator: greaterThan
+          left:
+            value:
+              complex:
+                root: NumOfOktaSuspiciousActivities
+            iscontext: true
+          right:
+            value:
+              complex:
+                root: inputs.OktaSuspiciousEventsThreshold
+            iscontext: true
+        - operator: greaterThan
+          left:
+            value:
+              complex:
+                root: NumOfFailedLogon
```

Comment:
What's the difference between 'NumOfFailedLogon' and 'AzureFailLoginCount' ?

Reply:
NumOfFailedLogon is an output from the Okta user investigation playbook, and AzureFailLoginCount is an output from the Azure user investigation playbook.